pom.xml

```diff
@@ -1749,6 +1758,7 @@
                     <encoding>utf-8</encoding>
                     <source>${java.source.version}</source>
                     <additionalJOption>${javadoc.options}</additionalJOption>
+                    <additionalparam>${javadoc.opts}</additionalparam>
```

Comment:
have you verified that this works with 1.7 as well, will the plugin error out because of the empty string value on 1.7?

Reply:
It does work on both Java7 & Java8 as the compiler. That's what I changed in 
d6c6186 when I realised I'd broken the compile with Java7